src/features/device-state/routes/state-get-v3.ts

```diff
@@ -270,7 +276,13 @@ const stateQuery = _.once(() =>
 		id: { uuid: { '@': 'uuid' } },
 		options: {
 			$select: ['device_name', 'os_version'],
-			$expand: deviceExpand,
+			$expand: {
+				...deviceExpand,
+				manages__device: {
+					$select: ['uuid', 'device_name'],
+					$expand: deviceExpand,
+				},
+			},
```

Comment:
Can you revert this - I'm pretty sure it's a failed rebase artifact
```suggestion
			$expand: deviceExpand,
```

Reply:
👍